lms/static/styles/components/_FilePickerApp.scss

```diff
@@ -31,9 +27,26 @@
   top: calc(50% - #{var.$spinner-size / 2});
 }
 
-.FilePickerApp__document-source-buttons {
-  align-items: stretch;
-  display: inline-flex;
-  flex-direction: column;
-  margin-top: 30px;
+.FilePickerApp__actions {
+  // A flex container (row) with two children: a column of buttons and a
+  // "spacer" to take up the rest of the row space not needed by the buttons.
+  // The spacer prevents the button container from stretching full width.
```

Comment:
I think "stretching full width" is meant to say "stretching full height"?

Reply:
The spacer is for width

```
Container text which is longer than the longest button
[button 1            ] [spacer.............................................
[b u t t o n 2       ]  
[b  u  t  t  o  n  3 ]                                                      ]
```

Reply:
Oh, right. I misread the first line that said "flex container (row)".